### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shannonpileggi**

### CODE REVIEW

**Comment by shannonpileggi** [comment]
some readme updates here, too

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+usethis::use_course("rstats-wtf/wtf-explore-libraries")
+```
+
+Choose your own adventure!
+
+* `01_explore-libraries_spartan.R` (directions to explore without suggested code)
+
+* `01_explore-libraries_comfy.R` (directions to explore with suggested code)
+
+* `01_explore-libraries_jenny.R` (directions to explore with code solutions)
```
Comment (jennybc) [inline README.md]: I wonder if we should replace "jenny" with "solution"? No strong opinion, but I offer it for your consideration.

**Review by jennybc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shannonpileggi) [reply README.md]: that makes sense, then agnostic to whoever teaches

File: 01_explore-libraries_solution.R [hunk 2]
```diff
@@ -6,11 +6,11 @@
 
 ## let's confirm the second element is, in fact, the default library
 .Library
-identical(.Library, .libPaths()[2])
```
Comment (jennybc) [inline 01_explore-libraries_solution.R:9]: I think this `[2]` is necessary for anyone who has a user library (presumably in position 1) followed by the default or (sort of) system library. This is the setup we basically recommend in WTF.

So I suspect there's a better edit here.

What's your motivation?

Comment (shannonpileggi) [reply 01_explore-libraries_solution.R:9]: Good catch, I didn't actually mean to edit that part, sorry! I was just running through the code on my personal set up. I'll correct.

**Pull request merged**